realm/realm-library/src/androidTest/java/io/realm/entities/migration/MigrationPosteriorIndexOnly.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2016 Realm Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.realm.entities.migration;
+
+import io.realm.RealmObject;
+import io.realm.annotations.PrimaryKey;
+import io.realm.migration.MigrationPrimaryKey;
+
+public class MigrationPosteriorIndexOnly extends RealmObject implements MigrationPrimaryKey {
```

Comment:
Add a comment about this class, e.g "Class used for testing what happens if you modify fields defined after the primary key field"